tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -45,12 +120,19 @@
               required=True, validators=[]),
         Field(item="73", name='RELATIONSHIP_HOH', type='number', startIndex=46, endIndex=48,
               required=True, validators=[]),
-        Field(item="74", name='PARENT_MINOR_CHILD', type='string', startIndex=48, endIndex=49,
-              required=True, validators=[]),
-        Field(item="75", name='EDUCATION_LEVEL', type='string', startIndex=49, endIndex=51,
-              required=True, validators=[]),
-        Field(item="76", name='CITIZENSHIP_STATUS', type='string', startIndex=51, endIndex=52,
-              required=True, validators=[]),
+        Field(item="74", name='PARENT_MINOR_CHILD', type='number', startIndex=48, endIndex=49,
```

Comment:
a handful of the `child_one` fields are missing validators present in `child_two`

```
GENDER
RECEIVE_NONSSA_BENEFITS
RECEIVE_SSI
RELATIONSHIP_HOH
PARENT_MINOR_CHILD
UNEARNED_SSI
OTHER_UNEARNED_INCOME
```